tests/Unit/Helpers/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -5,6 +5,8 @@
 
 #include <algorithm>
 #include <array>
+#include <boost/variant/multivisitors.hpp>
+#include <boost/variant/variant.hpp>
```

Comment:
why not use `std::variant`? I think it supports all the features that boost does, but slightly different names